.travis.yml

```diff
@@ -1,15 +1,18 @@
-sudo: false
 language: python
-python:
-  - "2.7"
+matrix:
+  allow_failures:
+    - python: "3.7"
+  include:
+    - python: "2.7"
+    - python: "3.7"
```

Comment:
Do you mind adding an entry for Python 3.6 while you're at it?  It's the default on Ubuntu 18.04, which we plan to upgrade to shortly, and we're also not sure if all of our dependencies have been updated yet to work with 3.7 (new `async` and `await` keywords, etc.).  Depending on how that unfolds, we might end up initially deploying on 3.6.  So for most of our repos, I think we should start testing both to see if there's a strong case for going with one version vs. the other.

Reply:
Done.  Running __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__ on the code will tell you if you are have any problems with the new `async` and `await` keywords.  However this assumes that there is not another syntax error that halts the linter first.  This is why my first move in each repo (#370) is to try to flatten all Python 3 syntax errors () as fast as possible.

Reply:
Running that [flake8](http://flake8.pycqa.org) command on the main EdX Python repos:
* https://travis-ci.org/cclauss/itinerant-tester/builds/479049721